mkinitcpio

```diff
@@ -400,6 +474,13 @@ while :; do
         -n|--nocolor)
             _optcolor=0
             ;;
+        -U|--uefi)
+            shift
+            [[ -d $1 ]] && die "Invalid image path -- must not be a directory"
+            if ! _optuefi=$(readlink -f "$1") || [[ ! -e ${_optgenimg%/*} ]]; then
+                die "Unable to write to path: \`%s'" "$1"
```

Comment:
Is there a reason the first quote here is a backtick?

Reply:
It's inherited from some other code. I have no clue.